spec/requests/users_spec.rb

```diff
@@ -27,12 +42,20 @@
     expect(json_body['email']).to eq(user.email)
   end
 
-  it 'should return 404 if user cannot be found' do
-    expected_response = { 'error' => 'User Not Found' }
+  it 'should return skateparks that user has favorited' do
+    user = create(:user)
+    park = create(:skatepark)
+    user.skateparks << park
 
+    get "/users/#{user.id}"
+
+    expect(json_body['skateparks'][0]['name']).to eq(park.name)
+    expect(json_body['skateparks'][0]['address']).to eq(park.address)
+  end
+
+  it 'should return 404 if user cannot be found' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.